README.md

```diff
@@ -69,73 +69,85 @@ Ensure that Nami is set to Testnet, that you have some Test Ada, and that you've
 
 ### Optional: Mint your own NFTs
 
-This process will be simplified in the future.
+UPDATE: see the minting section in `seabug-contracts/README.md` instead. The following minting process is currently broken, see [\#22](https://github.com/mlabs-haskell/seabug/issues/22).
```

Comment:
Keeping broken process in readme can be confusing even if it's marked as broken

Reply:
Unsure if its the right move to drop the full minting util, I feel like probably not, but we can certainly do so from the readme, and add back later once fixed

Reply:
The new minting process partially relies on `mint-nft.sh` to upload an image to IPFS+our database and get its base36 CID, and some of the setup from this section of the readme is necessary for that to work. It's kind of hacky right now, but I'm not sure if it's worth trying to extract out the part of `mint-nft.sh` that we need.

Maybe we just remove `mint-nft.sh` for now, but keeping the parts that are needed? The file is in git anyways

Reply:
See https://github.com/mlabs-haskell/seabug/pull/16/commits/43dad7d9f7572bd0833aadb32081fefb36241a39 for what I meant. I can revert that commit if we want to keep the old version, but I feel like it'll just cause confusion